Dockerfile

```diff
@@ -1,12 +1,19 @@
 ﻿FROM microsoft/dotnet:2.0-sdk
 WORKDIR /app
 
-# copy csproj and restore as distinct layers
-COPY LidarrAPI/*.csproj ./
-RUN dotnet restore
-
 # copy everything else and build
 COPY LidarrAPI/* ./
+COPY docker-services/LidarrAPI/docker-entrypoint.sh ./
+
+# There are some extra directories that cause issues for a build if they exist, remove them
+RUN rm -fr -- bin
```

Comment:
Might want to reconsider this since docker should start with a fresh image when building. If these directories are sticking around, it may be an indication of a different issue. If it does need to stay in, I think each `RUN` creates an intermediate container, so it may be better to do `RUN rm -rf bin obj Properties Debug`.

Reply:
If we are guarnteed that it will be a fresh instance each time.. I Think it's kosher to get rid of them..